components/armDecoder/encodings/armLoadStore.cpp

```diff
@@ -478,6 +478,10 @@ arminst LDP(armcode const &aFetchedOpcode, uint32_t aCPU, int64_t aSequenceNo) {
     /* Msutherl: if signed, only valid encodings are preindex, postindex, imm-index */
     return unallocated_encoding(aFetchedOpcode, aCPU, aSequenceNo);
   }
+  if (rt == rt2) {
+    // Constrain unpredictable
```

Comment:
Can we add a specific reference in the comment here about what behaviour is constrained unpredictable and a pointer to the manual? I think it would be hard to decipher this in the future otherwise.

Reply:
Yes, will do!